content/projects/oop/animals/part1/_index.md

```diff
@@ -178,20 +173,18 @@ home.make_all_sounds() # this doesn't return anything
 home.adopt_pet(dog1) # 1
 home.make_all_sounds()
 # this returns:
-# Bark
+# ["Bark"]
+
 
 home.adopt_pet(cat) # 2
 home.make_all_sounds()
 # this returns:
-# Bark
-# Meow
+# ["Bark", "Meow"]
 
 home.adopt_pet(dog2) # 3
 home.make_all_sounds
 # this returns:
-# Bark
-# Meow
-# Bark
+# ["Bark", "Meow", Bark]
```

Comment:
Might have missed adding the last index in a string, you have Bark, believe should be "Bark"